eCourse_backend/eCourse_backend/views.py

```diff
@@ -0,0 +1,19 @@
+from django.shortcuts import render, get_object_or_404
+from django.contrib.auth.decorators import permission_required, login_required
```

Comment:
please remove unneeded imports line 1 and permission_requierd

Reply:
should be resolved